### ISSUE TITLE
Wooden Wheels Can Break

### ISSUE THREAD

**Issue by Verycoolgarfield** [studio-minus/ppg-bugs-and-requests#1592]
**Feature description**
Wooden Wheels will break under pressure or by slamming against the floor or walls. Some fragments can pierce flesh!

**Feature purpose**
A bit more realism. Also since most other wooden items break

**Additional context**
![20220702074226_1~2](https://user-images.githubusercontent.com/108382993/177002173-7a756d45-e3ab-47c8-8ccb-94cfd3ef4d4e.jpg)

**Comment by DindinYT37** [comment]
There should be a separate wooden wheel that's breakable though, as this would break a lot of contraptions and generally - I think this would be a lot against ppg's "style" if it was suddenly being forced onto us that they're destructible.

**Comment by Verycoolgarfield** [comment]
> 

So like an option once you spawn it?

**Comment by DindinYT37** [comment]
> > 
> 
> So like an option once you spawn it?

No. We would have two wheels in the spawn-menu, one breakable and one unbreakable.

**Comment by Verycoolgarfield** [comment]
Ok that makes sense

**Comment by PossiblyRemember** [comment]
or maybe an option in the context menu to make destructible

**Comment by DindinYT37** [comment]
> or maybe an option in the context menu to make destructible

It's not machinery.

**Comment by PossiblyRemember** [comment]
> > or maybe an option in the context menu to make destructible
> 
> It's not machinery.

that's fine

**Comment by Verycoolgarfield** [comment]
I honestly just want it in the game as a separate object though

**Comment by PossiblyRemember** [comment]
> I honestly just want it in the game as a separate object though

that's fine, idrc how zooi adds it

**Comment by SomeBloodyMinger** [comment]
This would be a bad idea, function-wise. It would be hard to get it spinning without it breaking over time. Maybe there can be some 'breaking rules' like it can only be damaged by guns or weapons. Overall, I'm not sure.

**Comment by Verycoolgarfield** [comment]
I was thinking only being violently being pulled against the ground would break it. Normal rolling wouldn't.

**Comment by 69XAnonymousX69** [comment]
> There should be a separate wooden wheel that's breakable though, as this would break a lot of contraptions and generally - I think this would be a lot against ppg's "style" if it was suddenly being forced onto us that they're destructible.

or just make the destructible wheel indestructible using the context menu, how have you guys not thoughts about this.

**Comment by DindinYT37** [comment]
> > There should be a separate wooden wheel that's breakable though, as this would break a lot of contraptions and generally - I think this would be a lot against ppg's "style" if it was suddenly being forced onto us that they're destructible.
> 
> or just make the destructible wheel indestructible using the context menu, how have you guys not thoughts about this.

@PossiblyRemember literally said this 23 hours ago, and I said that it isn't machinery. How have you not read that?

**Comment by 69XAnonymousX69** [comment]
> > > There should be a separate wooden wheel that's breakable though, as this would break a lot of contraptions and generally - I think this would be a lot against ppg's "style" if it was suddenly being forced onto us that they're destructible.
> > 
> > 
> > or just make the destructible wheel indestructible using the context menu, how have you guys not thoughts about this.
> 
> @PossiblyRemember literally said this 23 hours ago, and I said that it isn't machinery. How have you not read that?

lol do you know that you can make wooden chairs and tables indestructible aswell? it doesn't have to machinery, there is nothing in the code that says it has to be machinery

**Comment by DindinYT37** [comment]
> > > > There should be a separate wooden wheel that's breakable though, as this would break a lot of contraptions and generally - I think this would be a lot against ppg's "style" if it was suddenly being forced onto us that they're destructible.
> > > 
> > > 
> > > or just make the destructible wheel indestructible using the context menu, how have you guys not thoughts about this.
> > 
> > 
> > @PossiblyRemember literally said this 23 hours ago, and I said that it isn't machinery. How have you not read that?
> 
> lol do you know that you can make wooden chairs and tables indestructible aswell? it doesn't have to machinery, there is nothing in the code that says it has to be machinery

You can't though? Also, I don't say it has to be machinery, it just feels wrong if it were; also you shouldn't assume something about code you know nothing about.

**Comment by 69XAnonymousX69** [comment]
> You can't though? Also, I don't say it has to be machinery, it just feels wrong if it were; also you shouldn't assume something about code you know nothing about.

why would it feel wrong, machinery isn't the only thing that can break 
you seem to have atleast something to say about every post, maybe stop being such a hypocrite

**Comment by DindinYT37** [comment]
> > You can't though? Also, I don't say it has to be machinery, it just feels wrong if it were; also you shouldn't assume something about code you know nothing about.
> 
> why would it feel wrong, machinery isn't the only thing that can break you seem to have atleast something to say about every post, maybe stop being such a hypocrite

It feels wrong because it does, there's no real explanation for that - it's just my opinion anyway and I think a destructible and indestructible version would be more "fitting" than having it as an option in the context menu. Also, I don't **have** to say something, I just do - I'm not being a hypocrite, I'm just pointing out the facts and give my own opinion.

Also, why did you skip the part where you said "do you know that you can make wooden chairs and tables indestructible aswell" without it being true, shouldn't you check something like that before using it in an argument?

**Comment by 69XAnonymousX69** [comment]
> > > You can't though? Also, I don't say it has to be machinery, it just feels wrong if it were; also you shouldn't assume something about code you know nothing about.
> > 
> > 
> > why would it feel wrong, machinery isn't the only thing that can break you seem to have atleast something to say about every post, maybe stop being such a hypocrite
> 
> It feels wrong because it does, there's no real explanation for that - it's just my opinion anyway and I think a destructible and indestructible version would be more "fitting" than having it as an option in the context menu. Also, I don't **have** to say something, I just do - I'm not being a hypocrite, I'm just pointing out the facts and give my own opinion.
> 
> Also, why did you skip the part where you said "do you know that you can make wooden chairs and tables indestructible aswell" without it being true, shouldn't you check something like that before using it in an argument?

i think you are being a hypocrite, you've been doing this since 2021 and i don't think you are aware of it
also im just saying what i remember, i might be incorrect and you can correct it and i wouldn't be a mad at you.
i can literally choose a random post and see your message.
why do you have to be so serious it's just a bug page for ppg, did zooi make you a mod or something? this is no place to for you to mini mod.

**Comment by PossiblyRemember** [comment]
bruh zooi will add it how he adds it, it won't affect it **at all**

**Comment by DindinYT37** [comment]
> i think you are being a hypocrite, you've been doing this since 2021 and i don't think you are aware of it also im just saying what i remember, i might be incorrect and you can correct it and i wouldn't be a mad at you. i can literally choose a random post and see your message. why do you have to be so serious it's just a bug page for ppg, did zooi make you a mod or something? this is no place to for you to mini mod.

I'm not a mod and not mini-modding, this is just how GitHub works. The whole point of this is being serious.

**Comment by Verycoolgarfield** [comment]
The honesty don't care how zooi adds it. I just wish it could be in the game.
